modules/users/client/views/admin/adduser.client.view.html

```diff
@@ -0,0 +1,44 @@
+<div>
```

Comment:
There are more fields that we need to get from the user including role and UFID. Ciannan has a code review up that takes all of these fields in if you want to look at that for guidance